tests/data/norovirus.mutations.vcf

```diff
@@ -0,0 +1,157 @@
+##fileformat=VCFv4.1
```

Comment:
Is this file important test data? It does not matter for the classification, does it?

Reply:
It is not, will remove it.